src/utils/components/SSHAccess/constants.ts

```diff
@@ -13,3 +13,5 @@ export enum SERVICE_TYPES {
 }
 
 export const METALLB_GROUP = 'metallb.io';
+
+export const exampleIdentityFilePath = '--identity-file=/home/jdoe/.ssh/id_rsa';
```

Comment:
What do you think of:  '--identity-file=~/.ssh/id_rsa'?

Reply:
this example was take from the cli help text, so I think it's better to stay synced